.github/workflows/tests.yaml

```
@@ -0,0 +1,236 @@
+      HEALTH_CHECK_TIMEOUT: 10
+      CHECK_INTERVAL: 5
+
+    steps:          
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build services with cache
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2035:info:2:9: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)